plugins/source/aws/codegen/recipes/base.go

```diff
@@ -82,6 +82,26 @@ func awsNameTransformer(f reflect.StructField) (string, error) {
 	return c.ToSnake(f.Name), nil
 }
 
+func awsResolverTransformer(f reflect.StructField, path string) (string, error) {
+	if f.Type.String() == "[]types.Tag" {
```

Comment:
Could we stick to the reflect data instead of string representations? (basically, better option will be to define names for key/value fields and then transform the resolver accordingly)

Reply:
Each "types" will be different, so it won't be a blanket-catch-all unless we go and import all of those different `types` from different packages.

Reply:
that's right. what I meant was using the info that there're only a couple of exposed fields with well-known names (or belonging to well-known names lists) to be used as key-value pairs